<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MedecinDialog</class>
 <widget class="QWidget" name="MedecinDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Médecins</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBoxMedecinForm">
     <property name="title">
      <string>Formulaire Médecin</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labelNom">
        <property name="text">
         <string>Nom :</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditNom"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelPrenom">
        <property name="text">
         <string>Prénom :</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditPrenom"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelSpecialite">
        <property name="text">
         <string>Spécialité :</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEditSpecialite"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelTelephone">
        <property name="text">
         <string>Téléphone :</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEditTelephone"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="labelEmail">
        <property name="text">
         <string>Email :</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="lineEditEmail"/>
      </item>
      <item row="5" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <!-- Bouton Ajouter avec icône -->
        <item>
         <widget class="QPushButton" name="btnAjouter">
          <property name="text">
           <string>Ajouter</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>:/icons/ajouter.png</normaloff>
           </iconset>
          </property>
         </widget>
        </item>
        <!-- Bouton Modifier avec icône -->
        <item>
         <widget class="QPushButton" name="btnModifier">
          <property name="text">
           <string>Modifier</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>:/icons/modifier.png</normaloff>
           </iconset>
          </property>
         </widget>
        </item>
        <!-- Bouton Supprimer avec icône -->
        <item>
         <widget class="QPushButton" name="btnSupprimer">
          <property name="text">
           <string>Supprimer</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>:/icons/supprimer.png</normaloff>
           </iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <!-- Table des Médecins -->
   <item>
    <widget class="QTableWidget" name="tableWidgetMedecins">
     <property name="columnCount">
      <number>6</number>
     </property>
     <property name="horizontalHeaderLabels" stdset="0">
      <stringlist>
       <string>ID</string>
       <string>Nom</string>
       <string>Prénom</string>
       <string>Spécialité</string>
       <string>Téléphone</string>
       <string>Email</string>
      </stringlist>
     </property>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
